libcontainer/cgroups/utils.go

```diff
@@ -130,10 +130,7 @@ func GetAllSubsystems() ([]string, error) {
 			}
 		}
 	}
-	if err := s.Err(); err != nil {
-		return nil, err
-	}
-	return subsystems, nil
+	return subsystems, s.Err()
```

Comment:
This looks to be a potential change in behavior, as before this change it would return `nil` in case of an error, and after this change, it would return `subsystems`